kin-recovery/recovery-sample/src/main/java/recovery/sample/AccountCreator.java

```diff
@@ -0,0 +1,87 @@
+package recovery.sample;
```

Comment:
isn't it code from `kin-sdk-sample`? can we share this code between the moudles?

Reply:
Yes it is and but a thinner version of the original code.
Maybe we need to think about some common code between the modules, or at least between the samples of all modules.